app/Config/Database.php

```diff
@@ -89,13 +89,13 @@ public function __construct()
 		{
 			$this->defaultGroup = 'tests';
 
-			// Under Travis-CI, we can set an ENV var named 'DB_GROUP'
+			// Under Github Actions, we can set an ENV var named 'DB'
```

Comment:
I would like to see this whole thing find a better home than **app/Config/Database.php**. I'm not sure where that is right now, but I will think on it.

Reply:
I think the way to do this will be with a Database Registrar in `Tests\Support` specific to this repo. Problem is that registrars are currently disabled during testing (I have no idea why - @lonnieezell?). I have opened https://github.com/codeigniter4/CodeIgniter4/pull/3693 to see about changing that.

Reply:
I don't have any recollection on why those were turned off for testing, and I can't think of a horrible reason to run them - so sounds go for it! :)